cmd/bridge/main.go

```diff
@@ -69,6 +76,7 @@ func main() {
 	fK8sModeOffClusterEndpoint := fs.String("k8s-mode-off-cluster-endpoint", "", "URL of the Kubernetes API server.")
 	fK8sModeOffClusterSkipVerifyTLS := fs.Bool("k8s-mode-off-cluster-skip-verify-tls", false, "DEV ONLY. When true, skip verification of certs presented by k8s API server.")
 	fK8sModeOffClusterPrometheus := fs.String("k8s-mode-off-cluster-prometheus", "", "DEV ONLY. URL of the cluster's Prometheus server.")
+	fK8sModeOffClusterThanos := fs.String("k8s-mode-off-cluster-thanos", "", "DEV ONLY. URL of the cluster's Prometheus server.")
```

Comment:
We should update the `oc-environment.sh` script to set this, see https://github.com/openshift/console/blob/master/contrib/oc-environment.sh#L20

Reply:
Actually... it looks like there's no route for Thanos. Is that correct? We use the Prometheus route today for our development environment where we run off cluster

Reply:
there is a new route for thanos querier which will be always available and we have to point the `Prometheus UI` link there now, see https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/thanos-querier/route.yaml